build.properties

```diff
@@ -37,4 +37,7 @@ re.version=3.0.0.342
 waila.version=1.5.8a
 wrcbe.version=1.4.1.2
 
+//EDIT_for_Flaxbeard's_Steam_Power_integration
+fsp.version=0.28.7
```

Comment:
Just insert it where it belongs in alphabetical order, no needs for the "EDIT"s, that's what we have Git for ;-)